+dj/Relvar.m

```diff
@@ -352,14 +352,25 @@ function update(self, attrname, value)
             
             switch true
                 case isNull
+                    assert(header.attributes(ix).isnullable, ...
+                            'attribute `%s` is not nullable.', attrname)
                     valueStr = 'NULL';
                     value = {};
                 case header.attributes(ix).isString
                     assert(dj.lib.isString(value), 'Value must be a string')
```

Comment:
Actually I don't think this will allow for inserting `[]` as `isString([])` would be false.

Reply:
Addressed